content/docs/Concepts/versioning.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Versioning"
+linkTitle: "Versioning"
+weight: 3
+date: 2020-04-10
+description: >
+  OpenCue policy on version number changes and compatibility between versions.
```

Comment:
Nit: remove the period for the description

Reply:
Done.